src/main.ts

```diff
@@ -1,21 +1,44 @@
 import * as core from "@actions/core";
-const { GitHub, context } = require("@actions/github");
+import { getOctokit, context } from "@actions/github";
 import parse from "parse-diff";
 import { rexify } from "./utils";
 
+async function getDiff(octokit, context) {
```

Comment:
What we are using here from context is payload.repository and payload.pull request. Maybe destructure  these in the argument declaration?